### PR TITLE
sklearnの課題完了しました。

### PR DESCRIPTION
**Pull request by joeyprg45**
なぜかaccuracyが１になっちゃったんですけど原因がわかりませんでした。

### CODE REVIEW

**Review by Ryoma0101 — COMMENTED** [review]
概ねいいですが、最後の機械学習モデルは**ランダムフォレスト**にしてください。
DecisionTreeClassifierは単一の決定木を用いた分類器で、ランダムフォレストではありません。
accuracy=1は正解率100%ということです。今回の課題では別に不思議な数字ではないので大丈夫です！
データフレームを確認する時`print(df)`を使うよりも`display(df)`の方が見やすいので、使ってみましょう！
[ランダムフォレストの使用方法など](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html)

**Review by Ryoma0101 — APPROVED** [review]
LGTMです。
　y_trainが1次元配列ではなく列ベクトルとして渡されているので警告が出ていますね。基本的に問題ないはずですが、sklearnでは1次元の行ベクトルを期待しているみたいなので、1次元配列に変換する方法も調べておいてください！
　今回は説明変数が少ない&指示通り課題をするだけなので特に問題なかったと思いますが、データコンペでは十数個の説明変数があることもあります。データを可視化して、モデリングの方法を考えることになると思うので、**EDA(探索的データ解析)**  やランダムフォレストの以外の**学習アルゴリズム**を調べておくといいと思います！
